src/http/kv/create_namespace.rs

```diff
@@ -0,0 +1,32 @@
+use cloudflare::apiclient::APIClient;
+use cloudflare::response::APIFailure;
+use cloudflare::workerskv::create_namespace as kv_api;
+
+use crate::settings::global_user::GlobalUser;
+use crate::settings::project::Project;
+
+use super::{api_errors_to_message, new_api_client, KvNamespace};
+
+pub fn create_namespace(
+    user: GlobalUser,
+    project: &Project,
+    title: String,
+) -> Result<KvNamespace, failure::Error> {
+    let client = new_api_client(user);
```

Comment:
It feels to me like wrapping `create_namespace` with an identically named function that has different arguments more pertinent to your abstractions ("project", "user") to be unnecessary boilerplate. There's very little substance in the implementation of this function. So I wouldn't do that.

But if I were doing that, I'd probably pass the `client` as an argument, to help make this more testable and less rigid.

And that would take away even more substance, making me question my choice of wrapping it even more.